tests/suites/model/destroy.sh

```diff
@@ -12,23 +12,27 @@ run_model_destroy() {
 	ensure "model-destroy" "${file}"
 
 	echo "Ensure current model is 'model-destroy'"
-	juju models --format json | jq -r '."current-model"' | check model-destroy
+	juju models --format json | jq -r '."current-model"' | check 'model-destroy'
 
 	echo "Add new model 'model-new'"
 	juju add-model model-new
 
 	echo "Ensure current model is 'model-new'"
-	juju models --format json | jq -r '."current-model"' | check model-new
+	juju models --format json | jq -r '."current-model"' | check 'model-new'
```

Comment:
minor: extra quotation doesn't really do anything, but maybe that's your style guide?